core/src/main/java/org/keycloak/TokenVerifier.java

```diff
@@ -292,7 +292,8 @@ private void removeCheck(Predicate<? super T> check) {
      * @param checks
      * @return
      */
-    public TokenVerifier<T> withChecks(Predicate<? super T>... checks) {
+    @SafeVarargs
```

Comment:
It seems that this modification is not relevant to DPoP support itself. It might be better to treat it by other issue and its PR.

Reply:
Agreed; please see #9555 / #9556